src/components/Employee/ManagerComponent/ManagerComponent.vue

```diff
@@ -0,0 +1,134 @@
+                v-if="managerObj.manager_id === img.manager_id"
+              >
+                <div style="display:flex">
+                  <img
+                    class="rounded-circle"
+                    :title="img.name"
+                    :src="managerObj.image"
+                    width="40"
+                    height="40"
+                  >
+                  <i
+                    class="fa fa-times-circle text-danger mt-4 close-collapse"
+                    aria-hidden="true"
+                    @click="closeCollapse(managerObj)"
+                  ></i>
+                  <span class="ml-3">
+                    <p class="text-primary fw-semi-bold fs-larger manager-name">{{img.name}}</p>
+                    <p class="text-dark manager-work">{{img.work}}</p>
+                  </span>
+                </div>
+                <div>
+                  <p class="mt-2 mb-2 text-dark">Manager Weight:</p>
+                  <ul class="progress-bar-employee">
+                    <li class="list" v-for="index in 10" :key="index">
+                      <div class="text-primary fw-semi-bold employee-progress d-flex">{{index}}</div>
+                    </li>
+                  </ul>
+                  <div class="all-manager-div">
+                    <div class="w-100">
+                      <p class="block-example mt-2">All</p>
+                    </div>
+                    <div style="display: flex">
+                      <div
+                        class="all-manager"
+                        v-for="img in managerShowing"
+                        :key="img.manager_id"
+                      >
+                        <img
+                          class="rounded-circle h-auto"
+                          v-b-tooltip.hover
+                          :title="img.name"
+                          :src="img.image"
+                          width="30"
+                        >
+                        <i class="fas fa-plus-circle fa-plus-circle-altered add-icon text-primary"></i>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </b-collapse>
+            </div>
+          <!-- </b-row> -->
+        </b-container>
+        </b-row>
+      </div>
+    </div>
+</template>
+
+<script>
+import { get, call, sync } from "vuex-pathify";
+
+export default {
+  name: 'ManagerComponent',
+  data() {
+    return { 
+      show: false,
+      managerObj: {}
+    }
+  },
+  computed: {
+    emp_arr: sync("manageEmployee/employees"),
+    managerShowing() {
+      let managers = []
+      this.emp_arr.forEach((element, index) => {
+        if (this.emp_arr[index].id === this.employeID) {
+          managers = this.emp_arr[index].manager 
+        }
+      })
+      return managers
+    }
+  },
+  props:{
+    manager: {type : Array, default: []},
+    employeID: {type: Number, default: 1}
+  },
+  methods:{
+    deleteManager: call("manageEmployee/deleteManager"),
```

Comment:
we cannot use pathify in components. components should be independent. they need to emit things. not directly update the state

Reply:
fix this everywhere